lib/Counter/styles.scss

```diff
@@ -0,0 +1,5 @@
+.counter {
+  font-size: 10px;
+  line-height: 9px;
+  min-width: 14px;
```

Comment:
How come we have fixed values like this?

Reply:
We should try getting these to the nearest REM value if they look ok. Otherwise we maybe need to align GCDS and the designs better?

Reply:
I've converted these to REM. They're fixed non-tailwind values because tailwind doesn't let me be really specific. This is such a tiny component (like 20px max) that there are just no useful helpers that give me that precision. Everything else was fine except for these values.